### DISCUSSION TITLE
Delete old user session on login

### DISCUSSION THREAD

**Discussion by sw-tt-jaydeepcharadva** [ExpressionEngine/ExpressionEngine#3519] [Feature Requests]
Hi,

I would like to add feature to delete old session from the sessions table. 

For example if one member of mysite is logged in with this url : **http://mylocal.org/admin.php?/cp/addons/settings/structure/index&S=10fd4af5045c3bf78154030acc3f607c** to the Expression engine and he is not logout, but remove the url upto the admin.php like **http://mylocal.org/admin.php**. 

So, as per the functionality he is directly logged out that's fine, but in this case the session id is not deleting from the database table sessions when he logged in again the duplicate entry of same users entered into the database which is security concerns i think. So, my suggestion is to delete the old sessions everytime, when user is logged in to the system.

Thank you

**Comment by intoeetive** [comment]
@sw-tt-jaydeepcharadva are you concerned with Session ID being present in URL? You have the option to change session type to "Cookies only" - in that case the URLs will look more "clean"

The old sessions are being cleaned up after some time of inactivity, but I admit that for some people the desire to delete the session records immediately after logout is legit.

Since this is a feature request, I am converting this to discussion - please use discussions for feature requests in the future. And if you think this is security concern and you know how to exploit it - please email details to support@expressionegine.com

Thanks!

**Reply by sw-tt-jaydeepcharadva** [reply]
Hi,

It's fine for us if its delete the session after some time of inactivity.

Thank you
